.github/workflows/deploy-gh-pages.yml

```diff
@@ -29,6 +29,8 @@ jobs:
       run: npm install
     - name: npm run build
       run: npm run build
+    - name: Check CNAME file existence
```

Comment:
We need to check CNAME before the push to avoid problems on the next step 🔢

Reply:
The current check step is already before the push step.
If the check fails, the push will be blocked.

Reply:
IMO, to avoid the domain takeover by GitHub again, we need to be more careful to make sure there is no intermediate operation between the check and push.

Reply:
BTW, we should remove the variables as possible which might cause unexpected cases in the future.